cea/demand/thermal_loads.py

```diff
@@ -1205,19 +1211,24 @@ def get_temperatures(locator, prop_HVAC):
     prop_emission_heating = pd.read_excel(locator.get_technical_emission_systems(), 'heating')
     prop_emission_cooling = pd.read_excel(locator.get_technical_emission_systems(), 'cooling')
     prop_emission_dhw = pd.read_excel(locator.get_technical_emission_systems(), 'dhw')
+    prop_system_vent = pd.read_excel(locator.get_technical_emission_systems(), 'ventilation')
+    prop_system_vent_ctrl = pd.read_excel(locator.get_technical_emission_systems(), 'ventilation_control')
 
     df_emission_heating = prop_HVAC.merge(prop_emission_heating, left_on='type_hs', right_on='code')
     df_emission_cooling = prop_HVAC.merge(prop_emission_cooling, left_on='type_cs', right_on='code')
     df_emission_dhw = prop_HVAC.merge(prop_emission_dhw, left_on='type_dhw', right_on='code')
+    df_system_ctrl_vent = prop_system_vent.merge(prop_system_vent_ctrl,left_on='code_ctrl', right_on='code', suffixes={'_v','_c'})
```

Comment:
here again, `df_system_ventilation_control` could also work (also, you switched around `vent` and `ctrl` between the `prop` and the `df`...